godot-macros/src/class/data_models/field.rs

```diff
@@ -41,6 +45,26 @@ pub struct Fields {
     /// The field with type `Base<T>`, if available.
     pub base_field: Option<Field>,
 
+    /// The base field is either absent or is correctly formatted.
+    ///
+    /// When this is false, there will always be a compile error ensuring the program fails to compile.
+    pub well_formed_base: bool,
+
     /// Deprecation warnings.
     pub deprecations: Vec<TokenStream>,
+
+    /// Errors during macro evaluation that shouldn't abort the execution of the macro.
+    pub errors: Vec<Error>,
```

Comment:
I usually tried to keep `venial` types qualified, since they often have very generic names (like here 🙂). 
```suggestion
    pub errors: Vec<venial::Error>,
```